Libplanet/Net/Swarm.cs

```diff
@@ -81,9 +89,14 @@ public partial class Swarm : ICollection<Peer>, IDisposable
 
             _distributeMutex = new AsyncLock();
             _receiveMutex = new AsyncLock();
+            _blockSyncMutex = new AsyncLock();
 
+            _ipAddress = ipAddress ?? GetLocalIPAddress();
+            _listenPort = listenPort;
+
+            string loggerId = _privateKey.PublicKey.ToAddress().ToHex();
             _logger = Log.ForContext<Swarm>()
-                .ForContext("Swarm_listenUrl", _listenUrl.ToString());
+                .ForContext("Swarm_id", loggerId);
```

Comment:
What's the convention?

Reply:
There is no convention at this time. 😢  I'll rename to `SwarmId`.